sdk/search/Azure.Search.Documents/src/Generated/AliasesRestClient.cs

```diff
@@ -54,7 +54,7 @@ internal HttpMessage CreateCreateRequest(SearchAlias @alias)
             request.Headers.Add("Accept", "application/json; odata.metadata=minimal");
             request.Headers.Add("Content-Type", "application/json");
             var content = new Utf8JsonRequestContent();
-            content.JsonWriter.WriteObjectValue(@alias);
+            content.JsonWriter.WriteObjectValue(@alias, ModelSerializationExtensions.WireOptions);
```

Comment:
This should be using this static [RequestContent.Create](https://github.com/Azure/azure-sdk-for-net/blob/2f8473ffda384d1010160fdbc552af69088b3471/sdk/core/Azure.Core/src/RequestContent.cs#L133) method.

Not sure if this blocks this PR, but for sure we want to open a follow up issue to track

/cc @JoshLove-msft just in case we have the same issue in azure emitter.